geoh5py/objects/object_base.py

```diff
@@ -323,11 +300,93 @@ def get_data(self, name: str) -> List[Data]:
 
     def get_data_list(self) -> List[str]:
         """
-        :return: List of names of data associated with the object
+        Get a list of names of all children :obj:`~geoh5py.data.data.Data`.
 
+        :return: List of names of data associated with the object
         """
         name_list = []
         for child in self.children:
             if isinstance(child, Data):
                 name_list.append(child.name)
         return sorted(name_list)
+
+    def get_property_group(
+        self, group_id: Union[str, uuid.UUID]
+    ) -> Optional[PropertyGroup]:
+        """
+        Retrieve a :obj:`~geoh5py.groups.property_group.PropertyGroup` from one of its
+        identifier.
+
+        :param uid: PropertyGroup identifier, either by its name or uuid
```

Comment:
missing period and obj reference
```suggestion
        :param uid: PropertyGroup identifier, either by its name or :obj:`uuid.UUID`.
```